pkg/provider/resource_cluster.go

```diff
@@ -168,61 +168,61 @@ func (c *ClusterResource) Schema() *schema.Resource {
 				Computed:    true,
 			},
 			"private_networking": {
-				Description: "Is private networking enabled",
+				Description: "Is private networking enabled.",
 				Type:        schema.TypeBool,
 				Optional:    true,
 			},
 			"cloud_provider": {
-				Description: "Cloud Provider",
+				Description: "Cloud Provider. For example, \"aws\" or \"azure\".",
 				Type:        schema.TypeString,
 				Required:    true,
 			},
 			"read_only_connections": {
-				Description: "Is read only connection enabled",
+				Description: "Is read only connection enabled.",
 				Type:        schema.TypeBool,
 				Optional:    true,
 			},
 			"region": {
-				Description: "Region",
+				Description: "Region to deploy the cluster. See https://www.enterprisedb.com/docs/biganimal/latest/overview/03a_region_support/ for supported regions.",
```

Comment:
```suggestion
				Description: "Region to deploy the cluster. See [Supported regions](https://www.enterprisedb.com/docs/biganimal/latest/overview/03a_region_support/.",
```

Reply:
Clickable link is added at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/0aea7392df32d1d3b7fcb6e35524aa585fa8a660